ExampleApp/ViewController.swift

```diff
@@ -234,33 +234,23 @@ class ViewController: UITableViewController {
             
             // save capture assets to photo library?
             imagePicker.captureSettings.savesCapturedPhotosToPhotoLibrary = savesCapturedAssets
-            
-            // presentation
-            // before we present VC we can ask for authorization to photo library,
-            // if we dont do it now, Image Picker will ask for it automatically
-            // after it's presented.
-            PHPhotoLibrary.requestAuthorization({ [unowned self] (_) in
```

Comment:
Ok, but, in this way it won't ask for permissions at all ?

Reply:
Image picker will ask for permissions, it was mentioned in comment :)

Reply:
well this will only remove asking for permissions in example app, I am not sure if it makes any sense to remove it from there.

Reply:
It was set correctly in image picker (except of missing collection view reload after gaining permissions), the only problem was in example app.